### PR TITLE
RATEST-350: Remove duplicate OCL module test

### PR DESCRIPTION
**Pull request by Piumal1999**
## Requirements

- [x] This PR has a title that briefly describes the work done, including the ticket number if there is a ticket.
- [x] I have tested the changes locally and verified that they work as intended.
- [x] I have added comments to my code where necessary to help others understand my changes.
- [x] I have read and agree to the Contributing Guidelines of this project.

## Summary

1. Removed OCL module test which was written with Cypress
2. Removed cypress since there were no other tests written with cypress

## Screenshots

*None.*

## Related Issue

https://issues.openmrs.org/projects/RATEST/issues/RATEST-350

## Other

This will also fix https://issues.openmrs.org/projects/RATEST/issues/RATEST-342

### CODE REVIEW

**Comment by kdaud** [comment]
I've added a comment on the slack conversation [here](https://openmrs.slack.com/archives/CP343JH1R/p1685058834161339?thread_ts=1685032232.951569&cid=CP343JH1R)

**Review by anjula-sack — APPROVED** [review]
Thank you! @Piumal1999

**Comment by kdaud** [comment]
@Piumal1999, which test case duplicates the one being omitted?

**Comment by Piumal1999** [comment]
> @Piumal1999, which test case duplicates the one being omitted?

RefApp 2.x Open Concept Lab E2E Test
https://github.com/openmrs/openmrs-contrib-qaframework/actions/workflows/refapp-2x-open-concept-lab.yml

**Comment by Piumal1999** [comment]
> @Piumal1999, have you taken a look at their respective feature files and identified that the user stories are the same?

Yeah @kdaud. These are the 2 feature files:

Selenium: https://github.com/openmrs/openmrs-contrib-qaframework/blob/master/qaframework-bdd-tests/src/test/resources/features/refapp-2.x/16-openConceptLab/openConceptLab.feature 

Cypress: https://github.com/openmrs/openmrs-contrib-qaframework/blob/master/qaframework-bdd-tests/src/test/resources/features/ocl/ocl-subscription-module.feature

**Comment by kdaud** [comment]
Are they testing the same functionality(user stories)?

**Comment by Piumal1999** [comment]
> Are they testing the same functionality(user stories)?

Yeah. The Cypress test checks only adding the subscription and importing.
The selenium test contains additional tests for editing the subscription and removing the subscription.

**Comment by kdaud** [comment]
Hi @Piumal1999, I see these tests are testing different features. The one written in selenium is testing out the OCL within the legacy ui and dealing with one system(server). The one in cypress is carrying out  system integration between two servers

My idea would be to leverage on migrating the cypress test to this repo → https://github.com/openmrs/openmrs-test-3refapp and have the module remain with selenium tests.

**Comment by Piumal1999** [comment]
Hi @kdaud Thanks for the response. I have some questions.

> the one in cypress is carrying out integration testing between two systems
Here, what do you mean by 'two systems'?

> Could be we can leverage on migrating the cypress test to this repo → https://github.com/openmrs/openmrs-test-3refapp
But that repo is used for Openmrs 3 tests. Isn't it?

**Comment by kdaud** [comment]
The cypress test is interacting with [refapp 2.x](https://github.com/openmrs/openmrs-contrib-qaframework/blob/master/qaframework-bdd-tests/src/test/resources/features/ocl/ocl-subscription-module.feature#L13) and [Dictionary manager app](https://github.com/openmrs/openmrs-contrib-qaframework/blob/d9e6398bf4a2c872a33fa06ae3a8fb3cd8e7a2eb/qaframework-bdd-tests/src/test/resources/features/ocl/ocl-subscription-module.feature#L5)

**Comment by Piumal1999** [comment]
> The cypress test is interacting with [2.x refapp](https://github.com/openmrs/openmrs-contrib-qaframework/blob/master/qaframework-bdd-tests/src/test/resources/features/ocl/ocl-subscription-module.feature#L13) and [Dictionary manager app](https://github.com/openmrs/openmrs-contrib-qaframework/blob/d9e6398bf4a2c872a33fa06ae3a8fb3cd8e7a2eb/qaframework-bdd-tests/src/test/resources/features/ocl/ocl-subscription-module.feature#L5)

Yeah. But [OpenMRS Dictionary manager](https://openmrs.staging.openconceptlab.org/login) is no longer there. So the test is also failing. OpenMRS dictionary manager was [deprecated in February 2023](https://openconceptlab.org/openmrs-dictionary-manager/) and they have suggested using the OCL Term browser instead.

File: qaframework-bdd-tests/package.json [hunk 1]
```diff
@@ -25,39 +24,28 @@
...
   "devDependencies": {
-    "@testing-library/cypress": "^8.0.0",
-    "@types/uuid": "3.4.7",
-    "cypress": "^9.5.3",
-    "cypress-cucumber-preprocessor": "^4.0.1",
-    "cypress-file-upload": "^5.0.8"
-  },
-  "cypress-cucumber-preprocessor": {
-    "step_definitions": "cypress/integration/cucumber/step_definitions"
+    "@types/uuid": "3.4.7"
```
Comment (kdaud) [inline qaframework-bdd-tests/package.json:45]: Do we still need this dependency?

Comment (Piumal1999) [reply qaframework-bdd-tests/package.json:45]: Oh right. We can remove that as well

Comment (Piumal1999) [reply qaframework-bdd-tests/package.json]: Fixed with [e205d86](https://github.com/openmrs/openmrs-contrib-qaframework/pull/486/commits/e205d863303e0a9a8d6a33b167a3d806206dc7de)

**Review by kdaud — APPROVED** [review]

**Pull request merged**